cns/cnsclient/cnsclient_test.go

```diff
@@ -225,7 +225,7 @@ func TestCNSClientRequestAndRelease(t *testing.T) {
 
 	addTestStateToRestServer(t, secondaryIps)
 
-	podInfo := cns.KubernetesPodInfo{PodName: podName, PodNamespace: podNamespace}
```

Comment:
We need to revert this and pass the way CNI passes it, this will KubernetesPodInfo only, and can we pass PodInterfaceIf and InfraContainerId in the ipConfigRequest.

It will be ideal to run this test for both old and new behaviour, but if we cant then at least move them to new as in production we are sure it will be new version